content/working-with-azure-key-vault-in-csharp/index.md

```diff
@@ -73,7 +73,7 @@ Managed identities are of the following two types:
     
 -   User-assigned - this is a stand-alone Azure resource that is created independent of the actual service instance. You can assign this identifier to one or more Azure resources simultaneously. As an example, you can set this identity to both an Azure Web App and an Azure Function at the same point in time.
 
-## **Create a Web App in the Azure Portal**
+## Create a Web App in the Azure Portal
 
 We will now create an Azure Web App instance with the permissions to access Azure Key Vault. Adhere to the steps given below that would guide you to create a new Azure Web App instance while you’re within the Azure portal:
```

Comment:
Please use contractions consistently throughout the blog. For example, `We will` vs. `We'll`; `You are` vs. `You're`

Reply:
Here, `alt-text` is missing for the images. Please add relevant `alt-text`, which should describe the image in a in 3 to 5 words.

Reply:
Use `--` to create an em dash here: In the Publish section, there are two choices, Code and Docker Container `-` choose Code

Reply:
Set the `operating` system to Windows